3-methyl-heptane/Wang_phi_0.5_m-h_0.0122_xO2_0.2075__xN2_0.3902_NUIG_RCM_.yaml

```diff
@@ -0,0 +1,161 @@
+---
+file-authors:
+    - name: Nathan Harms
+      ORCID: 0000-0003-2680-371X
+file-version: 0
+chemked-version: 0.4.1
+reference:
+    doi: 10.1016/j.proci.2012.06.001
+    authors:
+        - name: Weijing Wang
+        - name: Zhenhua Li
+        - name: Matthew A. Oehlschlaeger
+        - name: Darren Healy
+        - name: Henry J. Curran
+        - name: S. Mani Sarathy
+        - name: Marco Mehl
+        - name: William J. Pitz
+        - name: Charles K. Westbrook
+    journal: Proceedings of the Combustion Institute
+    year: 2013
+    volume: 34
+    pages: 335-343
+    detail: m-h_0.0122_xO2=0.2075_phi=0.5
+experiment-type: ignition delay
+apparatus:
+    kind: rapid compression machine
+    institution: National University of Ireland Galway
+    facility: rapid compression machine
+common-properties:
+    composition: &comp
+      kind: mole fraction
+      species:
+        - species-name: 3-methyl-heptane
+          InChI: 1S/C8H18/c1-4-6-7-8(3)5-2/h8H,4-7H2,1-3H3
+          amount:
+            - 0.0122 
+        - species-name: O2
+          InChI:  1S/O2/c1-2
+          amount:
+            - 0.2075 
+        - species-name: N2
+          InChI: 1S/N2/c1-2
+          amount:
+            - 0.3902 
+        - species-name: CO2
+          InChI: 1S/CO2/c2-1-3
+          amount:
+            - 0.3901
+    ignition-type: &ign 
+      target: pressure
+      type: d/dt max
+datapoints:  
+  - temperature:
+      - 373.0 kelvin
+    compressed_temperature:
```

Comment:
I believe we have a new(er) `rcm-data` field, see https://pr-omethe-us.github.io/PyKED/schema-docs.html#ignition-rcm-data All the RCM-specific data should go in that field